master/buildbot/newsfragments/db_schema_change_properties.bugfix

```diff
@@ -0,0 +1 @@
+Alters change properties from String to Text. Requires upgrade master. (:bug:`3197`)
```

Comment:
Please change to more user oriented message:

    Change properties can now be of unlimited size (previously was max 1024 bytes)